app/config/tstvhq/adventures.json

```diff
@@ -85,41 +85,4 @@
                     ]
                 }
             ]
-        },
-        {
-            "config": {},
-            "rooms": [ 
-               {
-                "id": "massAve",
-                "name": "MassAve",
-                "text": "You are walking along Mass Ave and see the NTB on the corner of Clarendon and Mass Ave. Do you keep walking or turn towards the glowing light away from Dunkin Donuts? Are you man or moth?",
-                "buttons": [
-                    {
-                        "text": "A mere man",
-                        "nextRoom": ["kitchen"]
-                    },
-                    {
-                        "text": "A mighty moth",
-                        "nextRoom": ["outsideClarendon"]
-                    }
-                ]
-            },
-            {
-                "id": "outsideClarendon",
-                "name": "Outside Clarendon",
-                "text": "You mighty moth, you! Are you going to head into Clarendon then? Or do you go ahead for another block or two and see if anyone is around at 24?",
-                "buttons": [
-                    {
-                        "text": "Clarendon",
-                        "nextRoom": ["clarendon"]
-                    },
-                    {
-                        "text": "Twenty Four Please",
-                        "nextRoom": ["twentyFour"]
-                    }
-                ]
-            }
-        ]
         }
-    ]
```

Comment:
this deletion looks wrong to me. did you try running this locally?

Reply:
This was @LEDFlashing -- Jack, do you need to make changes? Unclear from Andrew's comment.

Andrew, no, I haven't been able to run anything locally since the transition to VS Code.

Reply:
I certainly didn't. if you're saying I need to undo the deletion I'll just do it manually.

Reply:
err ... if I had edit access to that one.